core/plugins/org.polarsys.capella.core.ui.search/src/org/polarsys/capella/core/ui/search/result/CapellaSearchResultPage.java

```diff
@@ -28,13 +33,13 @@ public CapellaSearchResultPage() {
   }
 
   @Override
-  protected void elementsChanged(Object[] objects) {
-    getViewer().refresh();
+  protected void elementsChanged(Object[] elements) {
+    getViewerContentProvider().elementsChanged(elements);

```

Comment:
The **SearchResultPage** delegates the behavior to the appropriate **ContentProvider** (List or Tree).